src/queryCache.js

```diff
@@ -167,6 +167,8 @@ export function makeQueryCache() {
           throw err
         }
       }
+    } else {
+      return Promise.resolve(query.state.data)
     }
```

Comment:
Because this is async already, the `Promise.resolve` is redundant along with the else statement. I suggest this:
```suggestion
    }
    
    return query.state.data
```

Reply:
D'oh, I missed the fact that we're async already. Thanks!